File: PHybrid/lib_hybrid/src/main/java/com/prim/hybrid/entry/Setting.java
package com.prim.hybrid.entry;

import java.io.Serializable;

/**
 * @author prim
 * @version 1.0.0
 * @desc
 * @time 2/20/21 - 2:46 PM
 * @contact https://jakeprim.cn
 * @name PHybrid
 */
public class Setting implements Serializable {
    private String name;
    private String value;

    public Setting(String name, String value) {
        this.name = name;
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return "Setting{" +
                "name='" + name + '\'' +
                ", value='" + value + '\'' +
                '}';
    }
}

File: PrimFastKotlin/KotlinAction/app/src/main/java/com/prim/gkapp/Setting.kt
package com.prim.gkapp

import com.prim.gkapp.utils.Preference

/**
 * @desc
 * @author prim
 * @time 2019-05-26 - 10:48
 * @version 1.0.0
 */
class Setting {
    var email: String by Preference(AppContext, "email", "")
    var password: String by Preference(AppContext, "password", "")
}